src/test/java/io/github/azagniotov/metrics/reporter/cloudwatch/CloudWatchReporterTest.java

```diff
@@ -353,7 +370,7 @@ public void shouldReportCounterValueDelta() throws Exception {
     @Test
     public void shouldReportArithmeticMeanAfterConversionByDefaultDurationWhenReportingTimer() throws Exception {
         metricRegistry.timer(ARBITRARY_TIMER_NAME).update(1_000_000, TimeUnit.NANOSECONDS);
-        reporterBuilder.withArithmeticMean().build().report();
+        reporterBuilder.withArithmeticMean().withMeterUnitSentToCW(Milliseconds).build().report();
```

Comment:
What happens if the `withMeterUnitSentToCW(Milliseconds)` won't be called by the client code, what would be the behavior then?

Reply:
This would have sent the default unit.
Was changed in second Commit